aospy/test/test_objs/models.py

```diff
@@ -0,0 +1,19 @@
+from aospy.model import Model
+import runs
+
+
+am2 = Model(
```

Comment:
Here and elsewhere that has GFDL-specific files, we might want to put in some try/except guards such that these don't cause the overall aospy testing suite to crash from non-GFDL machines (thinking about this led me to open #62).

For the tests themselves, we can use unittest's [skip decorators](https://docs.python.org/2/library/unittest.html#unittest.skip).